docs/sources/datasources/loki/configure-loki-data-source.md

```diff
@@ -0,0 +1,129 @@
+---
+aliases:
+  - ../data-sources/loki/
+  - ../features/datasources/loki/
+description: Configure the Loki data source
+keywords:
+  - grafana
+  - loki
+  - logging
+  - guide
+menuTitle: Configure Loki
+title: Configure the Loki data source
+weight: 200
+---
+
+# Loki data source
```

Comment:
Is this the correct topic title? You've already got a topic with this name.

Reply:
changed to Configure the loki data source - thank you